### PR TITLE
Da 3294

### PR DESCRIPTION
**Pull request by lfpratik**

### CODE REVIEW

**Review by lukaszgryglicki — CHANGES_REQUESTED** [review]
When I follow `BUILD_IMAGES.md` and do:
1 - `cd ./docker-images/base/`.
and then
2 - `./collect_and_build.sh`.
I'm getting:
Screenshot 2021-01-08 at 07 08 54

Please fix this.

**Comment by lfpratik** [comment]
@lukaszgryglicki, I am not sure why you getting this error. Can you run more time

**Comment by lukaszgryglicki** [comment]
Let me see now.

File: docker-images/base/collect_and_build.sh [hunk 1]
```diff
@@ -24,7 +26,7 @@ git clone "https://github.com/AlDanial/cloc.git" --branch "1.88" --single-branch
 # revert slack api commit from perceval
 cd grimoirelab-perceval && git fetch origin pull/680/head:slack-history-api-revert && git checkout slack-history-api-revert && git rebase master && cd ..
 # rocketchat support
```
Comment (lfpratik) [inline docker-images/base/collect_and_build.sh:28]: cc: @lukaszgryglicki,

@linonymous, need your attention on line number 27.

Comment (lfpratik) [reply docker-images/base/collect_and_build.sh:28]: `cd grimoirelab-perceval && git fetch origin pull/680/head:slack-history-api-revert && git checkout slack-history-api-revert && git rebase master && cd ..`

Comment (lfpratik) [reply docker-images/base/collect_and_build.sh:29]: @lukaszgryglicki  and @linonymous, I fixed the issue. Can both of you take a look at it?

**Review by linonymous — APPROVED** [review]
/lgtm

**Comment by lukaszgryglicki** [comment]
Worked locally, I'll see if `p2o.py` and `dads` works from within an image built locally and then I'll merge.

**Review by lukaszgryglicki — APPROVED** [review]
Verified locally, the base image now builds, approving, but please do not merge, I'll merge when I also test a full SDS image locally.
/lgtm

**Comment by lukaszgryglicki** [comment]
Merged, I will deploy on the `test`

I won't do that for `prod` because updating p2o after so long time may break almost everything.

They usually change flags, logic, etc without changing the major version, just like they always are on 0.X.Y - so u never know what will happen.

Even if ALL works fine at the first glance, I’ll only apply to `test` and we can check `prod`, say, next week - if all is OK on the `test`. Makes sense?

cc @1010sachin @fayazg @lfpratik  ?

**Comment by lfpratik** [comment]
@lukaszgryglicki, I agree

**Pull request merged**